src/components/Faq/Faq.vue

```diff
@@ -0,0 +1,489 @@
+<template>
+  <div class="flex flex-col justify-center pt-20 pb-10 md:pt-28">
+    <p class="text-center text-4xl">Frequently Asked Questions</p>
+    <p class="text-center text-sm m-3">
+      If you don't find the answer to your questions, please contact us at
+      <a href="https://discord.gg/VwJp4KM" class="underline hover:text-blue-600"
+        >discord</a
```

Comment:
```suggestion
      If you don't find the answer to your questions, please contact any of our moderators via
      <a href="https://discord.gg/VwJp4KM" class="underline hover:text-blue-600"
        >Discord</a
```

Reply:
Hmm, I think everyone can answer, no need to be specific here, no need to contact mods as they will not report someone.

Reply:
Done 👍